### PR TITLE
Fixes #55: Ensure correct controller is used to render checkExport

### LINKED ISSUE #55: Nested gridfield compatibility seems broken
I think this may be an issue with nested grid fields.

Upon clicking the queued export button, the page reloads, and simply displays the string `SilverStripe\View\ViewableData_Customised`

This is occurring on ElementForm submission exports, and any other gridfield I've tried it on within elemental Elements.

### PR DESCRIPTION
**Pull request by dizzystuff**
Mimics GridFieldDetailForm_ItemRequest's getTopLevelController() method to make sure we're using the right controller in the stack to render the content of checkExport, rather than the gridfield's immediate parent controller.

### CODE REVIEW

**Review by emteknetnz — CHANGES_REQUESTED** [review]
Would you be able to retarget this at `2` - sorry the default branch was previously `master` when it should have been `2` - this has been changed at our end

**Comment by dizzystuff** [comment]
Done 👍

File: composer.json [hunk 1]
```diff
@@ -24,7 +24,10 @@
     "extra": {
         "expose": [
             "client"
-        ]
+        ],
+        "branch-alias": {
```
Comment (emteknetnz) [inline composer.json]: Remove the branch-alias

**Review by emteknetnz — CHANGES_REQUESTED** [review]
Seems fine, there's a few other instances in the code base where we do something similar

Just need to remove the branch alias

**Review by emteknetnz — APPROVED** [review]
Squash merge on green

**Comment by dizzystuff** [comment]
Any chance of merging this into a new release guys? (Please 🙂)

**Comment by emteknetnz** [comment]
@dizzystuff I've merged and tagged [2.6.2](https://github.com/silverstripe/silverstripe-gridfieldqueuedexport/releases/tag/2.6.2) for you

**Comment by dizzystuff** [comment]
@emteknetnz terrific, thanks!

**Pull request merged**